os/board/rtl8730e/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -305,6 +306,13 @@ static int rtk_drv_callback_handler(int type)
 	case 5:
 		trwifi_post_event(ameba_nm_dev_wlan0, LWNL_EVT_SOFTAP_STA_LEFT, NULL, 0);
 		break;
+	//LWNL_EVT_CONCURRENT_JOINED and LWNL_EVT_CONCURRENT_LEFT as placeholder
+	case 6:
+		trwifi_post_event(ameba_nm_dev_wlan1, LWNL_EVT_CONCURRENT_JOINED, NULL, 0);
```

Comment:
yes correct

Reply:
Hi @a-jhanwar , based on your application design, is there a need for driver to inform upper layer of LWNL_EVT_CONCURRENT_JOINED (wlan0 connected to AP, external device connected to wlan1 softAP)?

If it is not necessary, I think it should be sufficient to change case 3 and case 5 to post event to wlan1 instead of wlan0

Reply:
Let's merge this PR first to resolve the DHCP issue
We are also checking rtk_netmgr.c to see what further changes are needed for concurrent mode
If we find that case 6 and 7 is not necessary, we will remove it in future commits
Thanks~